cellarium/ml/models/common/decoder.py

```diff
@@ -0,0 +1,179 @@
+from typing import Literal
+from collections.abc import Iterable
+
+import torch
+from torch import nn
+
+from cellarium.ml.models.common.fclayer import FCLayers
+
+
+class DecoderSCVI(nn.Module):
```

Comment:
As a matter of principle, things that belong to `common` must be common :D `DecoderSCVI` is not common, because it is only used by SCVI. Either refactor it to something more general-purpose, or refactor out to `scvi.py`.